src/main/java/com/jfrog/ide/common/log/ProgressIndicator.java

```diff
@@ -1,9 +1,14 @@
 package com.jfrog.ide.common.log;
 
+import com.jfrog.xray.client.services.scan.ScanGraphProgress;
+
 /**
  * @author yahavi
  */
-public interface ProgressIndicator {
+public interface ProgressIndicator extends ScanGraphProgress {
```

Comment:
I found it a bit odd that the general ProgressIndicator extends the specific ScanGraphProgress.
Maybe it will be more clear if the ProgressIndicator will be used instead of the ScanGraphProgress (with the new necessary setFraction method)?

Reply:
You are right. I created a middle class connecting between the ide-plugins-common progress and xray-client-java progress.